modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -453,21 +453,23 @@ def accepted?(status)
            'Complete'].include? status
         end
 
-        def overdue?(tracked_item)
+        def overdue?(tracked_item, type)
+          return false unless type == 'wwsnfy'
+
           overdue_date_present?(tracked_item) ? tracked_item[:suspns_dt] < Time.zone.now : false
         end
 
         def overdue_date_present?(tracked_item)
           tracked_item[:suspns_dt].present? && tracked_item[:accept_dt].nil?
         end
 
-        def build_tracked_item(tracked_item, status, item, uploaded)
+        def build_tracked_item(tracked_item, status, item, uploaded, type)
```

Comment:
Instead of passing "type" here, could we just pass "overdue" instead, and set the value of overdue when building the individual tracked item category? This would match with the way we set "uploaded" currently. We can then set the default value of the "overdue" parameter to false, and only override for wwsnfy items.